.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Cilium",
+  "image": "quay.io/cilium/cilium-builder:5a32ee3ababf8b7b4dbe3d45280a2b00e5248053@sha256:aa90d2a3ecd98431c2efe58ab561fccb497969904d489f7a8f4d5ccffa3444e5",
```

Comment:
Will this keep getting updated when we bump builder images? I think part of that is automated now, just want to check that this will also keep up to date.

Reply:
Ah I see the update below :+1: